src/Config/Renderer.php

```diff
@@ -31,7 +31,7 @@ public static function render(Config $config, Vars $vars)
     {
         $template = $config->getTemplate();
 
-        if (!is_writable($template) || !is_readable($template)) {
+        if (!is_readable($template)) {
             throw new \RuntimeException(sprintf('The template file "%s" is either not readable, not writable or it doesn\'t exist.', $template));
```

Comment:
you should change the error message, now that you only check if the template is readable